compiler/src/dotty/tools/dotc/transform/Splicer.scala

```diff
@@ -21,9 +21,11 @@ import dotty.tools.dotc.util.Positions.Position
 import scala.reflect.ClassTag
 
 /** Utility class to splice quoted expressions */
-object Splicer {
+class Splicer {
```

Comment:
It's weird to make the whole thing a class if the only bit of state is the classloader cache used in a single method, the cache could be passed as an argument instead, and the classloader cache created